lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -669,7 +669,7 @@ void LLVMToSPIRV::transVectorComputeMetadata(Function *F) {
     auto ArgNo = I->getArgNo();
     SPIRVFunctionParameter *BA = BF->getArgument(ArgNo);
     if (Attrs.hasAttribute(ArgNo + 1, kVCMetadata::VCArgumentIOKind)) {
-      SPIRVWord Kind;
+      SPIRVWord Kind = SPIRVWord{};
```

Comment:
Will it work if we do `SPIRVWord Kind = {};` instead?
The same question for initializations below.

Reply:
Fixed